tutorials/xr/xr_action_map.rst

```diff
@@ -0,0 +1,246 @@
+
+The default action map
+======================
+
+Godot will automatically create a default action map if no action map file is found.
+
+.. warning::
+	This default map was designed to help developers port their XR games from Godot 3 to Godot 4. As a result this map essentially binds all known inputs on all controllers supported by default, to actions one on one. This is not a good example of setting up an action map but it does allow a new developer to have a starting point when they want to become familiar with Godot XR without designing a proper action map for their game first.
+
+For this walk through we're going to start with a blank action map. You can simply delete the "Godot action set" entry at the top by pressing the trash can icon and it will clear out all actions. You can also remove the controllers that you do not wish to setup, more on this later.
+
+Action sets
+===========
+
+.. note::
+  Before we dive in, you will see the term XR runtime used throughout this document. With XR runtime we mean the software that is controlling and interacting with the AR or VR headset and exposed to us through an API such as OpenXR. So for Steam this is SteamVR, for Meta on desktop this is the Oculus Client (including when using Quest link), for Meta on Quest this is the Quests native OpenXR client, on Linux this could be Monado, etc.
+
+The action map allows us to organise our actions in sets. These sets can be individually enabled/disabled.
+
+The concept here is that you could have different sets that provide bindings in different scenarios. You could have a set ``Character control`` for when you're walking around, ``Vehicle control`` for when you're operating a vehicle, ``Menu`` for when a menu is open, etc.
+Only the action set applicable to the current state of your game can then be enabled.
+
+This is especially important if you wish to bind the same input on a controller to a different action. For instance:
+
+ * in your ``Character control`` set you may have an action ``Jump``, 
+ * in your ``Vehicle control`` set you may have an action ``Accelerate``, 
+ * in your ``Menu`` set you may have an action ``Select``.
+
+All are bound to the trigger on your controller.
+
+OpenXR will only bind an input or output to a single action. If the same input or output is bound to multiple actions the one in the active action set with the highest priority will be the one updated/used. So in our above example it will thus be important that only one action set is active.
+
+For your first XR game we highly recommend starting with just a single action set and not over engineer things.
+
+For our walk through in this document we will thus create a single action set called ``my_first_action_set``, we do this by pressing the ``Add action set`` button:
```

Comment:
I do make an example like that in the paragraph (or two) above but I do agree that "my first action set" is a bit of a cheeky wink to the many "my first ..." tutorials, the screenshots are pesky to redo but it might be worth the change, not sure.